puppet/modules/chassis/manifests/php.pp

```diff
@@ -131,8 +89,6 @@
 					ensure => absent,
 				}
 			}
-			# "5.3",
-			# "5.4",
 			default: {
```

Comment:
We can actually remove this `case` now, as all versions now fall under the other case.